examples/deploy/terraform/cluster/main.tf

```diff
@@ -83,6 +83,19 @@ module "external_deployments_operator" {
   external_deployments = var.external_deployments_operator
 }
 
+module "flyte" {
+  count                     = var.flyte.enabled ? 1 : 0
+  source                    = "./../../../../modules/flyte"
```

Comment:
This feels fragile, is there any way we can use [path expressions](https://developer.hashicorp.com/terraform/language/expressions/references#filesystem-and-workspace-info) instead?

Reply:
there is a script that handles the module source. https://github.com/dominodatalab/terraform-aws-eks/blob/main/examples/deploy/set-mod-version.sh.